gno.land/pkg/gnoland/genesis.go

```diff
@@ -187,3 +188,29 @@ func LoadPackage(pkg gnomod.Pkg, creator bft.Address, fee std.Fee, deposit std.C
 
 	return tx, nil
 }
+
+func LoadEmbeddedStdlibs(deployer crypto.Address, fee std.Fee) []TxWithMetadata {
+	stdlibs := stdlibs.EmbeddedMemPackages()
+	stdlibsTxs := []TxWithMetadata{}
+
+	for _, memPkg := range stdlibs {
+		if memPkg.Path == "testing" {
+			continue
+		}
+
+		tx := TxWithMetadata{Tx: std.Tx{
+			Fee: fee,
+			Msgs: []std.Msg{
+				vmm.MsgAddPackage{
+					Creator: deployer,
+					Package: memPkg,
+				},
+			},
+		}}
+
+		tx.Tx.Signatures = make([]std.Signature, len(tx.Tx.GetSigners()))
```

Comment:
This is gonna be touchy with Miguel's genesis tx signature PR

Reply:
I'll wait on it before trying to merge this one

Reply:
actually he told me that we keep disabled signatures in genesis so this should be okay